src/components/templates/Audience/Audience.tsx

```diff
@@ -235,22 +250,51 @@ export const Audience: React.FunctionComponent = () => {
   const rowsForSizedAuditorium = minRows + auditoriumSize * 2;
   const columnsForSizedAuditorium = minColumns + auditoriumSize * 2;
 
+  // 3 because 1/3 of the sie of the auditorium, * 2 because we're calculating in halves due to using cartesian coordinates + Math.abs
+  const carvedOutWidthInSeats = Math.ceil(columnsForSizedAuditorium / (3 * 2));
+
+  // Keep a 16:9 ratio
+  const carvedOutHeightInSeats = Math.ceil(carvedOutWidthInSeats * (9 / 16));
+
+  // Calculate the position/size for the central video container
+  const videoContainerWidthInSeats = carvedOutWidthInSeats * 2 + 1;
+  const videoContainerHeightInSeats = carvedOutHeightInSeats * 2 + 1;
+  const videoContainerTopOffsetInSeats = Math.floor(
+    (rowsForSizedAuditorium - carvedOutHeightInSeats * 2) / 2
+  );
+  const videoContainerLeftOffsetInSeats = Math.floor(
+    (columnsForSizedAuditorium - carvedOutWidthInSeats * 2) / 2
```

Comment:
[OOS] Not sure how I feel about this(magical numbers). I do appreciate the comments about these numbers, but they still appear to me somewhat magical. Maybe we should move them into consts?

Reply:
I'm not necessarily against consts for them, though figured it would make it overly verbose, particularly in the case where it is just doubling/halving a thing.

I see this is a temporary stop-gap, I feel like the refactor should be able to come up with a much cleaner implementation than this, as it feels overly convoluted/hacky to me still (even though it seemed to work)